megamek/src/megamek/common/Board.java

```diff
@@ -607,19 +610,43 @@ public boolean contains(int x, int y) {
      * @param c the Coords.
      * @return <code>true</code> if the board contains the specified coords
      */
-    public boolean contains(Coords c) {
+    public boolean contains(@Nullable Coords c) {
         if (c == null) {
             return false;
         }
         return contains(c.getX(), c.getY());
     }
 
     /**
-     * @param c the Coords, which may be null
+     * Returns the Hex at the given Coords, both of which may be null.
+     *
+     * @param coords the Coords to look for the Hex
      * @return the Hex at the specified Coords, or null if there is not a hex there
      */
-    public @Nullable Hex getHex(final @Nullable Coords c) {
-        return (c == null) ? null : getHex(c.getX(), c.getY());
+    public @Nullable Hex getHex(final @Nullable Coords coords) {
+        if (coords == null) {
+            LogManager.getLogger().warn("Method called with null Coords!");
```

Comment:
Just gotta warn you, this will slow down processing in "performance sensitive areas" everything even if the debug level is set so that the logging won't occur - so I don't know if we actually want to be doing this logging.

Reply:
Yeah, removed it. Looked good on paper.